### PR TITLE
chore(update): update styling files for both templates

### PR DESCRIPTION
**Pull request by PatrickDohn**
#### Summary

- Work Type:  Chore 

#### What I did

- updated styling names
- condensed styling files for templates to single files for each template
-

#### Checklist

- [x] Branch is in format `TYPE/scope/description`
- [x] PR title is in format `TYPE(scope): description`
- [x] Commits are in semantic format
- [x] Has Summary
- [x] Has Labels

### CODE REVIEW

File: src/components/pages/MahomesPage.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { MahomesTemplate } from 'src/templates/MahomesTemplate';
-import data from '../../example-json/john_smith.json';
+import data from '../../example-json/patrick_dohn.json';
```
Comment (davidholyko) [inline src/components/pages/MahomesPage.js:3]: revert this

File: src/components/templates/FolesTemplate/Styles.js [hunk 2]
```diff
@@ -33,13 +31,53 @@ export const styles = StyleSheet.create({
...
+    fontSize: 24,
+    fontWeight: 'bold',
+  },
+  'info-header-title': {
+    fontSize: 14,
+  },
+  'info-header-location': {
+    fontSize: 14,
+  },
+  'info-content-header': {
```
Comment (davidholyko) [inline src/components/templates/FolesTemplate/Styles.js:70]: this classname is different than the others

Comment (davidholyko) [reply src/components/templates/FolesTemplate/Styles.js:70]: maybe `info-header-contact`?

File: src/components/templates/FolesTemplate/Styles.js [hunk 3]
```diff
@@ -33,13 +31,53 @@ export const styles = StyleSheet.create({
...
-  'description-item': {
-    padding: '0 0 0 0',
-    margin: '0 0 2px 8px',
+  'section-heading': {
+    fontSize: 14,
+    margin: '8px 0',
+    textAlign: 'center',
+    borderBottom: 1,
   },
   'project-content--heading': {
```
Comment (davidholyko) [inline src/components/templates/FolesTemplate/Styles.js:40]: maybe we want to rename this one that has the double dash

File: src/components/templates/FolesTemplate/Styles.js [hunk 4]
```diff
@@ -33,13 +31,53 @@ export const styles = StyleSheet.create({
...
+    margin: '0 12px 4px 0',
+  },
+  'info-header': {
+    padding: '0 0 0 24px',
+  },
+  'info-header-name': {
+    fontSize: 24,
+    fontWeight: 'bold',
+  },
+  'info-header-title': {
```
Comment (davidholyko) [inline src/components/templates/FolesTemplate/Styles.js:64]: this and `info-header-title` and `info-header-location` are actually the same thing, maybe we can find a name that fits both

File: src/components/templates/MahomesTemplate/Styles.js [hunk 5]
```diff
@@ -68,4 +83,36 @@ export const styles = StyleSheet.create({
...
+    padding: '0 0 0 12px',
+    width: '35%',
+  },
+  'info-heading': {
+    fontSize: 16,
+    fontWeight: 'bold',
+    margin: '8px 0 8px 0',
+    padding: '0 0 0 0px',
+  },
+  'list-item': {
```
Comment (davidholyko) [inline src/components/templates/MahomesTemplate/Styles.js:98]: maybe prepend this with the section it belongs to. `info-list-item`?

File: src/components/templates/FolesTemplate/Styles.js (hunk 3 above)
Comment (davidholyko) [reply src/components/templates/FolesTemplate/Styles.js:40]: `project-heading`? if that makes sense

**Review by davidholyko — APPROVED** [review]
LGTM

File: src/components/templates/FolesTemplate/Styles.js (hunk 4 above)
Comment (PatrickDohn) [reply src/components/templates/FolesTemplate/Styles.js:59]: I agree I just don't know a good name. Do you have any ideas?

Comment (davidholyko) [reply src/components/templates/FolesTemplate/Styles.js:59]: not sure. I dont want to hold this PR up for an idea. You can leave a TODO comment.

```
// TODO: consolidate this with the style with the same properties of font bold
```

Either that or `font-lg` lg stands for large

**Pull request merged**